node/src/GlideClusterClient.ts

```diff
@@ -314,6 +318,151 @@ export class GlideClusterClient extends BaseClient {
         );
     }
 
+    /**
+     * @internal
+     */
+    protected scanOptionsToProto(
```

Comment:
Move to `commands.ts`

Reply:
Why? It's for cluster scan only, I don't want it in a generic scope.